netlify.toml

```diff
@@ -5,8 +5,3 @@
 [build]
   publish = "dist"
   command = "npx pnpm i --store=node_modules/.pnpm-store && npx pnpm run build"
-
-[[redirects]]
-  from = "/*"
-  to = "/index.html"
```

Comment:
Not sure about removing this, would this breaks dynamic routes for SPA?

Reply:
oh...It will but not now, because 404 page use index.html's javascript file so it will get 404.html then render the dynamic route's page.

Usually each page should use its own preload Links(like js or css file), but we haven't resolved that in wite-ssg so every page is using index.js, than index.js will resolve the route to get correct files.

Reply:
In nuxt.js they use 404 page as a fallback page, which handle 404 and dynamic routes in SPA mode.